src/mongocxx/test/private/write_concern.cpp

```diff
@@ -23,7 +23,6 @@ using namespace mongocxx;
 
 TEST_CASE("creation of write_concern passes universal parameters to c-driver's methods",
           "[write_concern][base][c-driver]") {
```

Comment:
I don't get why this line went away. If this line went away, I'd expect hundreds of others to also go away. Can you check that the script didn't fall over half way through? There are other formatting issues that I expected to see fixed by this commit.